packages/gatsby-source-wordpress/src/__tests__/routePatternExamples.js

```diff
@@ -0,0 +1,17 @@
+const minimatch = require(`minimatch`)
+
+describe.only(`Glob patterns in the README`, () => {
```

Comment:
```suggestion
describe(`Glob patterns in the README`, () => {
```

Reply:
Also generally, not really sure what these tests are testing. It looks like we're basically testing minimatch?

Reply:
As minimatch is used within `checkRouteList` I just wanted to (programmatically) show that the suggested patterns would work on both types of WordPress installations. Also as a reference point for futures issues/questions regarding the exclude/include patterns. If more examples/bugs come up, we could extend the tests.

Reply:
And thank you for spotting the `.only`. In other projects I used to have a pre-commit hook for this, spotting `.only` within tests ... ;)

I corrected it.

Reply:
@riddla cool! But generally - we want to test _our own_ code, otherwise updating our code won't make this test fail (ever).

So we would want to actually write some assertions that validate `checkRouteList` and import that function, or a function that invokes it, in order to test this more effectively.

Reply:
Yeah, you're right. After all it's just about the examples in the Readme and not the `checkRouteList` itself. I removed the test and just added the information that it carried to the commit message. Should be sufficient for future reference, if this topic ever comes up again in some issue.